source/cl/source/extension/source/extension.cpp.in

```diff
@@ -253,6 +260,85 @@ cl_int GetObjectDetailExtensionInfoSingleValue(
   return retcode;
 }
 
+/// @brief Query all passed in extensions via the get_info_fn functor/lambda
```

Comment:
perhaps this could say more e.g. helper function to assist with querying blah, by providing it with a functor .....

Reply:
Yeah.. Again, just copying the other APIs in this file. Should I fix them all up? It can be done afterwards, before, but it might cloud the PR.

Reply:
I'll see how quick it'd be to resolve this properly across all APIs, and put up a PR later on.